docs/tutorial/index.md

```diff
@@ -354,53 +202,46 @@ ssl_public_key = C:\Users\<YOUR_WINDOWS_USER_NAME>\landscape_server.pem
+Click on the "Continue" button and you will see a status screen confirming the configuration is complete.
 
 ![Configuration is complete](./assets/status-complete.png)
 
-Done! You can close the UP4W window if you want.
+Done! You can now close the UP4W window.
 
-This has attached your Ubuntu Pro subscription to UP4W on the Windows host; UP4W will automatically forward it to the Ubuntu Pro client on your Ubuntu WSL instances; thus, all of your Ubuntu WSL instances will be automatically added to your Ubuntu Pro subscription.
+Your Ubuntu Pro subscription is now attached to UP4W on the Windows host.
+UP4W will automatically forward the subscription to the Ubuntu Pro client on your Ubuntu WSL instances.
+This means that all Ubuntu WSL instances will be automatically added to your Ubuntu Pro subscription.
 
 This has also configured the Landscape client built into your UP4W Windows agent to know about your Landscape server; UP4W will forward this configuration to the Landscape client on your Ubuntu WSL instances as well; and all systems where the Landscape client has been configured this way are automatically registered with Landscape.
 
+### UP4W host registration
 
-### Approve your UP4W Windows host registration with Landscape
-
-Go back to the Windows web browser and refresh the Landscape page. On the right-hand side of the main content area of the
+Go back to your web browser and refresh the Landscape page at `http://127.0.0.1:8080`. On the right-hand side of the
 page you should see a request to approve your Windows host registration ("Computers needing authorisation").
-Click on the computer name (below, `mib`); then, when the new page loads, click **Accept**.
-
-```{note}
-**If you've already closed the browser tab**:
+Click on the computer name (in this case: `mib`) and when the new page loads click **Accept**.
 
-Just open a new one, navigate to `http://127.0.0.1:8080` and log in with the credentials you created earlier.
-```
 ![Approve the Windows host registration](./assets/host-pending-approval.png)
 
 ![Accept host](./assets/accept-host.png)
 
-At the top of the page, on the right-hand side of the Landscape logo, click on "Computers". You should see your host machine listed there. Details such as the operating system may take a few minutes to appear.
+At the top of the page, on the right-hand side of the Landscape logo, click on "Computers". You should see your host machine listed. Details such as the operating system may take some time to appear.
 
 ![Host and WSL instances in Landscape](./assets/host.png)
 
-All set. From now on you can use your Landscape server not just to manage the Ubuntu WSL instances that UP4W has pro-attached and Landscape-registered on the host, but to tell UP4W to create and provision them on the host too!
-
+Now you can leverage UP4W from your Landscape server to create and provision Ubuntu WSL instances on the host.
 
-## Watch UP4W transform your Ubuntu Pro game on WSL!
+## Deploy WSL instances
```

Comment:
No strong opinions here: Teodora's idea with this heading was to make it somewhat exciting, click-baity, more informal and less technical/cold. Did you consider this change in the documentation mood when doing this change?

Reply:
The basic issue with the "transform your game" phrase is that it is unclear. When someone quickly scans a table of contents they want to know where to find the information they need. This phrase assumes that the user knows that their Ubuntu Pro game needs transforming in the first place. Then it doesn't immediately suggest how the product will transform their game.

I am sympathetic for the need to add some personality to docs; however, this doc currently needs a lot of editing, clarifying and restructuring. During that process these quirks will be a distraction, will quickly become out-of-place and - worse - could be annoying to a user. A quirky/informal phrase will not make a tutorial less technical/cold if the tutorial is already highly complex and technical. There will be opportunities to add personality again but I think it's less of a priority right now.

Ultimately what I would like to aim for is personality that makes sense in the context of the product. Instead, of saying to the user that they can "transform their game" it would be good to show that we as a team are aware of the pain and difficulty faced by those doing admin for large fleets of devices and how this product can make their life easier.